### ISSUE TITLE
can we get method comments using soot?

### ISSUE THREAD

**Issue by penelope24** [soot-oss/soot#1460]
As we know soot use compiled class files as input, however I wonder if we can still manage to get the comments of a method which are in the source files.

**Comment by MarcMil** [comment]
No. Comments are not present in Java class files, and thus there is no way to retrieve them when you only have the class file. You need the source code.
